src/components/LeftSidebar/InvitationHandler.vue

```diff
@@ -51,6 +51,7 @@
 					</NcButton>
 				</li>
 			</ul>
+			<NcLoadingIcon v-if="invitationsLoadedCount === 0" :size="20" />
```

Comment:
There could an edge case where you have an invitation and then the inviter removes it while you have the handler open, after fetch it will remain loading.